src/System.Windows.Forms.Design.Editors/src/System/ComponentModel/Design/CollectionEditor.cs

```diff
@@ -2033,9 +2033,8 @@ object ICustomTypeDescriptor.GetPropertyOwner(PropertyDescriptor pd)
             }
 
             /// <summary>
-            ///  ListItem class. This is a single entry in our list box.
-            ///  It contains the value we're editing as well as accessors for the type
-            // converter and UI editor.
+            ///  This is a single entry in our list box. It contains the value we're editing as well
+            /// as accessors for the type converter and UI editor.
```

Comment:
The rest of the lines have the wrong indentation, but this one should at least match.

```suggestion
            ///  as accessors for the type converter and UI editor.
```